src/i18n/translations/en.json

```diff
@@ -219,7 +225,8 @@
     "copy_push_token": "Copy push token",
     "current_theme": "Current theme: {{theme}}",
     "sign_out": "Sign out!",
-    "selected": " - selected"
+    "selected": " - selected",
+    "remove_account": "Remove account"
```

Comment:
the same string added to `common`. Please follow alphabetical order in this file